apps/skolplattformen-app/translations/en.json

```diff
@@ -1,18 +1,18 @@
 {
-    "abbrevations": {
+    "abbreviations": {
```

Comment:
The changes to this file would need to be applied to its siblings in order to not break the world.